現地：依存関係が足りてない

```
[INFO] [WallTime: 1494468087.129001] Loading controller: position_trajectory_controller
[ERROR] [1494468087.130764255]: Could not load controller 'position_trajectory_controller' because controller type 'position_controllers/JointTrajectoryController' does not exist.
[ERROR] [1494468087.130789140]: Use 'rosservice call controller_manager/list_controller_types' to get the available types
[ERROR] [WallTime: 1494468088.131273] Failed to load position_trajectory_controller
[INFO] [WallTime: 1494468088.132087] Controller Spawner: Loaded controllers: joint_state_controller

```

--- a/tra1_bringup/package.xml
+++ b/tra1_bringup/package.xml
@@ -14,6 +14,7 @@
   <build_depend>tra1_description</build_depend>
 
   <run_depend>controller_manager</run_depend>
+  <run_depend>joint_trajectory_controller</run_depend>
   <run_depend>joint_state_controller</run_depend>
   <run_depend>position_controllers</run_depend>
   <run_depend>robot_state_publisher</run_depend>

